packages/types/src/schemas/schemas.ts

```diff
@@ -9381,7 +9381,7 @@ export const MultiValueViewConfigSchema = {
 	]
 } 
 
-export const DashboardItemConfigSchema = {
+export const DashboardReportConfigSchema = {
```

Comment:
Did these changes come from the dev branch? If not, won't they break code in other packages?

Reply:
That was a mistake from me 🤦